faculty_cli/cli.py

```diff
@@ -53,7 +53,7 @@
 import faculty_cli.update
 import faculty_cli.version
 import faculty_cli.templates
-
+from faculty_cli import templates
```

Comment:
This module is already imported above and this shorter name is only used once (AFAICT) - can we remove this import line and just use it as `faculty_cli.templates` below?